content/en/service_management/incident_management/_index.md

```diff
@@ -95,17 +95,13 @@ Once you have the [Datadog integration enabled on Slack][7], from any Slack chan
 
 In the creation modal, you add a descriptive title, select whether customers were impacted (yes, no, or unknown) and select a severity level (1-5, unknown).
 
-If the user declaring the incident has connected their Slack to their Datadog account, then, by default, that user will become the Incident Commander. The Incident Commander (IC) can be changed later in-app if necessary. If the person declaring an incident is not a member of a Datadog account, then the IC is assigned to a generic `Slack app user` and can be assigned to another IC in-app.
+If the user declaring the incident has connected their Slack to their Datadog account, then, by default, that user becomes the Incident Commander. If the person declaring an incident is not a member of a Datadog account and anonymous Slack declaration is enabled in incident settings, then the IC becomes unassigned. The Incident Commander (IC) can be changed on the [incidents page][1] later if necessary.
```

Comment:
i don't think it makes sense to call it "anonymous Slack declaration" here since the setting is called 
"Allow users to declare incidents without a connected Slack account". i think we could leave this out of this section maybe since the incident can't be declared if this setting is off(is that right?)

Reply:
The second point is true. I think the main issue is that the setting is never mentioned anywhere so it reads to me that the incident _is_ declared regardless of that setting.

Reply:
okay fair. maybe the settings part should go [here](https://docs.datadoghq.com/service_management/incident_management/incident_settings#slack-features) though? like this it reads kind of like the IC being unassigned has something to do with that setting, not the actual incident getting declared

Reply:
Agree. I'll go ahead and leave it out of this copy though and won't mention it in my original copy to prevent an endless feedback loop as most of this documentation is already stale.